zerver/views/documentation.py

```diff
@@ -91,7 +91,7 @@ def get_path(self, article: str) -> DocumentationArticle:
         endpoint_name = None
         endpoint_method = None
 
-        # Absolute path cases
+        # Absolute path cases for help and policies documentation
         if (self.policies_view or self.help_view) and self.path_template.startswith("/"):
```

Comment:
Does the `path_template.startswith` check here do anything? I ask mainly because it makes it feel pretty hard to think about in what situations the final else statement below will apply.

Reply:
It did do something here because there are cases where the policies documentation may be in `templates/`, which will be a relative path. I updated this `get_path` function to consider that specific case first with a comment so that it's a bit clearer why we're checking for relative paths and loading templates.